__tests__/app.test.js

```diff
@@ -22,4 +22,40 @@ describe("NC_Games API", () => {
         });
     });
   });
+  describe("GET /api/reviews/:review_id", () => {
+    it("200: Returns all reviews for the given review_id", () => {
+      return request(app)
+        .get("/api/reviews/2")
+        .expect(200)
+        .then(({ body }) => {
+          const { review } = body;
+          expect(review).toHaveProperty("review_id", expect.any(Number));
+          expect(review).toHaveProperty("title", expect.any(String));
+          expect(review).toHaveProperty("review_body", expect.any(String));
+          expect(review).toHaveProperty("designer", expect.any(String));
+          expect(review).toHaveProperty("review_img_url", expect.any(String));
+          expect(review).toHaveProperty("votes", expect.any(Number));
+          expect(review).toHaveProperty("category", expect.any(String));
+          expect(review).toHaveProperty("owner", expect.any(String));
+          expect(review).toHaveProperty("created_at", expect.any(String));
+          expect(Date.parse(review.created_at)).not.toBeNaN();
+        });
+    });
+    it("400: bad request, when review id is valid but doesn't exist", () => {
```

Comment:
This would be 404 - 100000 is a valid number, there just isnt an article with an id of that